tactics/hints.ml

```diff
@@ -720,7 +718,7 @@ struct
   let remove_list env grs db =
     let filter (_, h) =
       match h.name with PathHints [gr] -> not (List.mem_f GlobRef.equal gr grs) | _ -> true in
-    let hintmap = GlobRef.Map.map (remove_he db.hintdb_state filter) db.hintdb_map in
```

Comment:
Note that this was a bug *en puissance*, because had we taken the transparent state of the `add` function instead of the `lookup` one to build the dnet, it would have been wrong in this call.